showpass/Showpass-E2E-tests
# TA-48 verify purchased tickets labels and buttons

**Description:**
This spec will be verifying that previously purchased event tickets can be viewed on the ‘Upcoming’ page and that the ticket details are correct.
IMPORTANT: event end day determines if the event is supposed to show up on the ‘Upcoming’ page

Steps:

- Log into the portal as 'test-user@mailinator.com' https://beta.showpass.com/account/upcoming/
- Focus only on the Event-1 and Event-2
- Open the purchased tickets
- Verify that tickets contain the correct details

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-48

**How to test:**
Run the following test suite and ensure all specs pass.
cypress/e2e/user-settings/upcoming.cy.js

**Test execution results:**

<img width="1085" alt="TA-48-pass" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/cbf0759a-b98d-4e82-b477-e96e69c4912b">
